docs/lib/graphqlapi/fragments/android/getting-started.md

```diff
@@ -1,41 +1,17 @@
-You can integrate with Amplify framework using the following steps:
+> ### Prerequisites
+> * An Android project targeting Android API level 16 (Android 4.1) or above.
+> * [Install and configure](~/cli/start/install.md) the Amplify CLI
 
-1. Setup the API endpoint and authentication information in the client side configuration.
-2. Generate Java Model classes from the API schema.
-3. Write app code to run queries, mutations and subscriptions.
+## GraphQL API with Amplify
 
-## Set up your backend
+The Amplify GraphQL API category provides a solution for making HTTP requests to GraphQL endpoints.  It supports [AWS AppSync](https://aws.amazon.com/appsync/) or any other GraphQL backend.
```

Comment:
This actually isn't true at the moment. We only support AWS AppSync, I believe.

Reply:
@jpignata - I was just chatting with @nikhil-dabhade about this and he pointed that the JS libraries do support other GQL backends as well. Is this Android-specific?

Reply:
Yes, JavaScript does, but neither iOS or Android do at the moment, I believe.

Reply:
I actually copied this from the JS page, so good to know that iOS and Android don't support it yet :) Will remove.